tests/vmi_hook_sidecar_test.go

```diff
@@ -303,17 +303,24 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 
 		Context("with ConfigMap in sidecar hook annotation", func() {
 
-			It("should update domain XML with SM BIOS properties", func() {
+			DescribeTable("[test_id:9833]should update domain XML with SM BIOS properties", func(withImage bool) {
 				cm, err := virtClient.CoreV1().ConfigMaps(testsuite.GetTestNamespace(vmi)).Create(context.TODO(), RenderConfigMap(), metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				vmi.ObjectMeta.Annotations = RenderSidecarWithConfigMap(hooksv1alpha2.Version, cm.Name)
+				if withImage {
```

Comment:
Using a boolean like this is not a good practice.
Booleans as arguments to a function/method are a smell, although sometimes it makes sense.

Here it does not make sense and you better pass the render function as input instead.
```go
DescribeTable("[test_id:9833]should update domain XML with SM BIOS properties", func(generateAnnotation func(string, string) bool) {
  ...
  vmi.ObjectMeta.Annotations = generateAnnotation(hooksv1alpha2.Version, cm.Name)
  ...
},
  Entry("when sidecar image is specified", RenderSidecarWithConfigMapPlusImage),
  Entry("when sidecar image is not specified", RenderSidecarWithConfigMapWithoutImage),
)
```